web/src/views/User/UserDetailPage.vue

```diff
@@ -89,6 +108,44 @@ export default defineComponent({
             :authenticated="false"
             :width="24"
           />
+          <v-row>
+            <v-col
+              cols="2"
+              class="pt-6"
+            >
+              <v-text-field
+                v-model="user.email"
+                label="Email"
+                dense
+                filled
+                :rules="requiredRules('E-mail is required',[
+                  v => /.+@.+\..+/.test(v) || 'E-mail must be valid',
+                ])"
+                @blur="updateUser($event.target.value)"
+              >
```

Comment:
Do we want to maybe wrap the `updateUser` call with some logic that ensures the email is valid? I seem to be able to save my user with a nonsense email:

![image](https://github.com/user-attachments/assets/ef9321ca-0c20-40de-a7bf-f915a6703213)